src/components/CoinsList.js

```diff
@@ -4,14 +4,19 @@ import { fetchCoins } from '../redux/feature/coinSlice';
 import CoinsCard from './coinsCard';
 
 const CoinsList = () => {
-  const coins = useSelector((store) => store.coin.coins);
-
+  const { coins, isLoading } = useSelector((store) => store.coin);
```

Comment:
* As stated in the [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=list%20of%20categories%20that%20could%20be%20filtered%20out%20by%20some%20parameter%2C%20for%20example%20by%20the%20category%20name.) your HOME page should have filter functionality whereby a list of categories could be filtered out by some parameter, for example by the category name. Kindly add this functionality to your app

Reply:
- [ ] Good job adding the filter function as requested, however, you need to implement this functionality on the home page, meaning you need to add a search bar where users can actually filter content by the given parameter using the function you just created.

e.g
![image](https://github.com/As1imwe-Mark/Crypto-Hub/assets/81932478/a6e0892d-032f-4a7b-8880-3fcd915b420a)